CalibFormats/SiStripObjects/interface/SiStripHashedDetId.h

```diff
@@ -29,6 +29,9 @@ class SiStripHashedDetId {
   /** Copy constructor. */
   SiStripHashedDetId(const SiStripHashedDetId &);
 
+  /** Assignment operator. */
+  const SiStripHashedDetId &operator=(const SiStripHashedDetId &) { return *this; }
```

Comment:
@aandvalenzuela are you sure this is correct?

Reply:
No, I am not. This is what the code format reported (see https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-43423/37938/code-format.patch). I was actually taking this issue offline...

Reply:
I mean - are you sure you have implemented `operator=` correctly?

Reply:
I think that is the default implementation, but I am not familiar with it. Do you have any suggestions?

Reply:
Does something like this look better?
```
TSinglePedEntry& operator=(const TSinglePedEntry& orig) {
    m_pedestalSqSum = orig.m_pedestalSqSum;
    m_pedestalSum = orig.m_pedestalSum;
    m_entries = orig.m_entries;
    return *this; 
 }
```